platform/src/main/java/org/stellar/anchor/platform/controller/sep/PlatformController.java

```diff
@@ -45,7 +45,16 @@ public PatchTransactionsResponse patchTransactions(@RequestBody PatchTransaction
   @RequestMapping(
       value = "/transactions",
       method = {RequestMethod.GET})
-  public GetTransactionResponse getTransactions() throws AnchorException {
-    throw new NotFoundException("Not implemented");
+  public GetTransactionsResponse getTransactions(
+      @RequestParam(value = "sep") TransactionsSeps sep,
+      @RequestParam(required = false, value = "order_by", defaultValue = "created_at")
+          TransactionsOrderBy order_by,
+      @RequestParam(required = false, value = "order", defaultValue = "asc") Sort.Direction order,
+      @RequestParam(required = false, value = "statuses") List<SepTransactionStatus> statuses,
+      @RequestParam(required = false, value = "limit", defaultValue = "200") Integer limit,
+      @RequestParam(required = false, value = "cursor", defaultValue = "0") Integer cursor)
+      throws AnchorException {
+    return transactionService.getTransactionsResponse(
```

Comment:
I think you're passing in `limit` for `pageNumber` and `cursor` for `pageSize`, right?

Also, I'm not sure we want to call our page number `cursor`. It will be easier to understand if we just call it `page`, wdyt?

Reply:
Yeah i forgot to rename parameters. 
page_number and page_size should be good, right?